lib/v2/bilibili/radar.js

```diff
@@ -143,5 +143,13 @@ module.exports = {
                 target: '/bilibili/live/room/:roomID',
             },
         ],
+        show: [
+            {
+                title: 'bilibili会员购票务',
```

Comment:
[Do not](https://docs.rsshub.app/joinus/new-radar#title) repeat the website name (`_name`).
```suggestion
                title: '会员购票务',
```